sdks/versions.mk

```diff
@@ -23,7 +23,9 @@ ANDROID_SDK_VERSION_x86_64?=21
 
 # iOS
 
-XCODE_DIR?=/Applications/Xcode.app/Contents/Developer
+XCODE_DIR?=/Applications/Xcode101.app/Contents/Developer
```

Comment:
I would keep the directory name to Xcode.app and follow the convention that it is always the latest stable Xcode (10.1 in the current situation)

Reply:
Okay, I've moved the settings needed for the bots to `run-jenkins.sh`. Is that better?

Reply:
what if we make `Xcode.app` point to the latest one and `Xcode32bit.app` to the one which can build x86 apps? then we don't need to hardcode version numbers in the scripts anywhere and just rely on bots being set up correctly.

Reply:
we can do that later. for now, if we would do that, it would break current master.

Reply:
Not if you add another symlink for the old name, but fair enough :)